### DISCUSSION TITLE
apache ignite support

### DISCUSSION THREAD

**Discussion by semsaksoy** [jeremyevans/sequel#2061] [General]
Hello

apache ignite uses jdbc thin client, but I couldn't activate it with the continuation, is there any way to use it?

**Comment by jeremyevans** [comment]
Do you have a working JDBC connection string for it?  If so, you should be able to pass it to `Sequel.connect`, assuming you are using JRuby.  Note that Sequel doesn't have any ignite-specific SQL support.
